app/controllers/landlords_controller.rb

```diff
@@ -0,0 +1,36 @@
+class LandlordsController < ApplicationController
+  def create
+  	if current_user.type == 'Landlord'
+	  	landlord = Landlord.new(landlord_params)
+	  	landlord.save!
```

Comment:
what happens if we aren't able to save the record (there's a database issue), we should check for this

https://github.com/calblueprint/RMI/blob/master/app/controllers/api/answers_controller.rb